client/payment-methods/index.js

```diff
@@ -261,6 +261,9 @@ const PaymentMethods = () => {
 											getStatusAndRequirements( id )
 												.status
 									}
+									// The card payment method is required when UPE is active and it can't be removed.
```

Comment:
I agree that allowing the `card` payment method to be disabled is strange (we have a warning pop-up, though). But since I made sure that when UPE is active, the default state is with the `card` active, I don't see a situation where the merchant would get stuck.

When UPE is not active, the merchant can check/uncheck the `card` at will.
When UPE is active, the `card` will always be checked regardless of the previous value.

Am I missing something?

Reply:
I've checked it again, it works as you described. It was probably an issue on my end while doing some changes to test different scenarios.

So everything good here 🙂